src/include/config-h.in.cmake

```diff
@@ -336,4 +335,7 @@
 /* Defined if boost::context is available */
 #cmakedefine HAVE_BOOST_CONTEXT
 
+/* Define if libradosstriper is enabled: */
```

Comment:
@chardan change "Define" to "Defined" here?

Reply:
The style in the file isn't entirely consistent-- but I'll go with "defined".

Reply:
This doesn’t appear to have happened.